### PR TITLE
Update workflow

### PR DESCRIPTION
**Pull request by jrbourbeau**

### CODE REVIEW

File: pipeline/reduce.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 
 @task
-def save_query(region, part_type):
+def save_query(segment):
```
Comment (mrocklin) [inline pipeline/reduce.py:21]: Maybe a better name?  Something that's connected to the query itself like "revenue_by_supplier" or something?

File: pipeline/monitor.py [hunk 2]
```diff
@@ -1,9 +0,0 @@
-import requests
-from prefect import flow
-
-
-@flow
-def check_model_endpoint():
-    r = requests.get("http://0.0.0.0:8080/health")
-    if not r.json() == ["ok"]:
-        raise ValueError("Model endpoint isn't healthy")
```
[the remark this answers is not in the document]
Comment (jrbourbeau) [reply pipeline/monitor.py:9]: Yes. Not added yet -- that's for me to do today

File: pipeline/reduce.py [hunk 3]
```diff
@@ -30,109 +30,55 @@ def save_query(region, part_type):
...
-            outfile = RESULTS_DIR / region / part_type / "result.snappy.parquet"
+            ).compute()
+            outfile = RESULTS_DIR / f"{segment}.snappy.parquet"
             fs.makedirs(outfile.parent, exist_ok=True)
             result.to_parquet(outfile, compression="snappy")
 
 
 @flow
 def query_reduce():
     with lock_compact:
```
[the remark this answers is not in the document]
Comment (jrbourbeau) [reply pipeline/reduce.py:81]: I went with `unshipped_orders_by_revenue` -- further suggestions are welcome

File: pipeline/reduce.py (hunk 1 above)
Comment (jrbourbeau) [reply pipeline/reduce.py]: I went with ``unshipped_orders_by_revenue`` -- further suggestions are welcome

**Pull request merged**